src/test/java/core/basesyntax/db/StorageTest.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.db;
+
+import java.util.Map;
+import java.util.Set;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class StorageTest {
+    private Storage fruitStorage;
+
+    @BeforeEach
+    void setUp() {
+        fruitStorage = new Storage();
+    }
+
+    @Test
+    void check_fruitStorage_Empty_OK() {
+        Assertions.assertEquals(0, fruitStorage.getData().size());
+    }
+
+    @Test
+    void check_fruitStorage_Update_OK() {
+        fruitStorage.updateFruitQuantity("banana", 50);
+        Set<Map.Entry<String, Integer>> storageData = fruitStorage.getData();
+        for (Map.Entry<String, Integer> testData : storageData) {
+            Assertions.assertEquals(50, testData.getValue());
+            Assertions.assertEquals("banana", testData.getKey());
+        }
+        fruitStorage.updateFruitQuantity("banana", 50);
+        for (Map.Entry<String, Integer> testData : storageData) {
+            Assertions.assertEquals(100, testData.getValue());
+            Assertions.assertEquals("banana", testData.getKey());
+        }
+
+    }
```

Comment:
using static imports on commonly used method can make code cleaner:
```suggestion
import org.junit.jupiter.api.Test;

import static import org.junit.jupiter.api.Assertions.assertEquals;

public class StorageTest {
    private Storage fruitStorage;

    @BeforeEach
    void setUp() {
        fruitStorage = new Storage();
    }

    @Test
    void check_fruitStorage_Empty_OK() {
        assertEquals(0, fruitStorage.getData().size());
    }

    @Test
    void check_fruitStorage_Update_OK() {
        fruitStorage.updateFruitQuantity("banana", 50);
        Set<Map.Entry<String, Integer>> storageData = fruitStorage.getData();
        for (Map.Entry<String, Integer> testData : storageData) {
            assertEquals(50, testData.getValue());
            assertEquals("banana", testData.getKey());
        }
        fruitStorage.updateFruitQuantity("banana", 50);
        for (Map.Entry<String, Integer> testData : storageData) {
            assertEquals(100, testData.getValue());
            assertEquals("banana", testData.getKey());
        }

    }
```